docs/env-definitions.md

```diff
@@ -10,7 +10,7 @@ If you're on version `>=3.8.0` you can make use of the `env` property in [flow-t
 
 ```json
 {
-  env: ["jsx", "node", ...],
+  "env": ["jsx", "node", ...],
```

Comment:
I left the `...` as it was, even though it's invalid JSON, because I figured that is intentional.

However, the key being quoted already reduces the amount of red that GitHub is displaying here.